backend/app/api/v1/llm.py

```
@@ -35,7 +24,7 @@ async def create_report(
         medical_text: str = Form("", description="Medical history + symptoms"),
         enable_llm_judge: bool = Form(False, description="Run LLM-as-judge after report generation"),
         # files
-        ct_images: List[UploadFile] = File(..., description="One ZIP archive or multiple CT images PNG/JPEG"),
+        ct_images: UploadFile = File(..., description="ZIP archive with CT images"),
```

Ruff (0.15.14)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)